azimuth.js

```diff
@@ -365,6 +365,30 @@ async function getPoint(contracts, point, what) {
   return data;
 }
 
+/**
+ * Get a list of unspawned/spawnable points
+ * @param {Object} contracts - An Urbit contracts object.
+ * @param {Number} point - Point number.
+ * @return {Promise<Array<Number>>} - Unspawned children of point
+ */
+async function getUnspawnedChildren(contracts, point) {
+  let size = getPointSize(point);
+  if (size >= PointSize.Planet) {
+    return [];
+  }
+  let spawned = await getSpawned(contracts, point);
+  let unspawned = [];
+  let childCount = (size === PointSize.Galaxy) ? 255 : 65535;
```

Comment:
Check my back-of-the-seat index math, but It looks like:

* `childCount` could be 256 or 65536 respectively
* `stepSize` could be eliminated and replaced with `childCount` in the remainder of the body
* the loop end condition could be changed to `i < childCount`

Just simplifies things somewhat, unless I've made an arithmetic error that I should be embarrassed by.

Reply:
> childCount could be 256 or 65536 respectively

Sure, `2^8` and all that, but remember that `0` doesn't count! (Because `0x0000` is equal to `0x00`.)

Reply:
Never mind, I was interpreting this incorrectly. The code can indeed be simpler!